src/NuGet.Core/NuGet.Packaging/Signing/CertificateUtility.cs

```diff
@@ -20,19 +20,19 @@ public static class CertificateUtility
         /// </summary>
         /// <param name="cert">X509Certificate2 to be converted to string.</param>
         /// <returns>string representation of the X509Certificate2.</returns>
-        public static string X509Certificate2ToString(X509Certificate2 cert)
+        public static string X509Certificate2ToString(X509Certificate2 cert, string indentation = "")
         {
             var certStringBuilder = new StringBuilder();
-            X509Certificate2ToString(cert, certStringBuilder);
+            X509Certificate2ToString(cert, certStringBuilder, indentation);
             return certStringBuilder.ToString();
         }
 
-        private static void X509Certificate2ToString(X509Certificate2 cert, StringBuilder certStringBuilder)
+        private static void X509Certificate2ToString(X509Certificate2 cert, StringBuilder certStringBuilder, string indentation)
         {
-            certStringBuilder.AppendLine(string.Format(Strings.CertUtilityCertificateSubjectName, cert.Subject));
-            certStringBuilder.AppendLine(string.Format(Strings.CertUtilityCertificateHash, cert.Thumbprint));
-            certStringBuilder.AppendLine(string.Format(Strings.CertUtilityCertificateIssuer, cert.IssuerName.Name));
-            certStringBuilder.AppendLine(string.Format(Strings.CertUtilityCertificateValidity, cert.NotBefore, cert.NotAfter));
+            certStringBuilder.AppendLine($"{indentation}{string.Format(Strings.CertUtilityCertificateSubjectName, cert.Subject)}");
+            certStringBuilder.AppendLine($"{indentation}{string.Format(Strings.CertUtilityCertificateHash, cert.Thumbprint)}");
+            certStringBuilder.AppendLine($"{indentation}{string.Format(Strings.CertUtilityCertificateIssuer, cert.IssuerName.Name)}");
+            certStringBuilder.AppendLine($"{indentation}{string.Format(Strings.CertUtilityCertificateValidity, cert.NotBefore, cert.NotAfter)}");
```

Comment:
String format needs to use CurrentCulture for user facing strings (this applies everywhere)